RosettaJdbi3/src/test/java/com/hubspot/rosetta/jdbi3/TestDao.java

```diff
@@ -12,6 +14,10 @@ public interface TestDao extends SqlObject {
   @SqlQuery("SELECT * FROM test_table")
   List<TestObject> getAll();
 
+  @SqlQuery("SELECT * FROM test_table")
+  @KeyColumn("id")
+  Map<Integer, TestObject> getAllMap();
```

Comment:
Should this be `Map.Entry` to test the code change? Or is this serving more as a regression test?